Project.toml

```diff
@@ -50,7 +52,6 @@ StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 UnicodePlots = "b8865327-cd53-5732-bb35-84acbb429228"
 VisualRegressionTests = "34922c18-7c2a-561c-bac1-01e79b2c4c92"
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 
 [targets]
-test = ["BinaryProvider", "Pkg", "Test", "Random", "StatsPlots", "VisualRegressionTests", "LaTeXStrings", "Images", "ImageMagick", "RDatasets", "FileIO", "UnicodePlots"]
```

Comment:
BinaryProvider is still required in the tests to download the reference images. That's why they are failing on travis.

Reply:
I don't mind making the change, but are you sure this is the case? I was under the impression that this section was for test-only dependencies, and dependencies of Plots were available already. For example, `GeometryTypes`  is used in the tests and is a dependency of Plots, but is not declared in `[targets]`. Additionally, the specific Travis error is:
```
curl: (77) Error reading ca cert file /etc/pki/tls/certs/ca-bundle.crt - mbedTLS: (-0x3E00) PK - Read/write of file failed
ERROR: LoadError: LoadError: failed process: Process(`curl -g -L -f -o /home/travis/build/JuliaPlots/Plots.jl/test/reference_images/reference_images.zip https://github.com/JuliaPlots/PlotReferenceImages.jl/archive/v0.0.1.tar.gz`, ProcessExited(77)) [77]
```
which seems like it might not be BinaryProvider-related.

Reply:
I think the tests use BinaryProvider's `download` and that's why they are failing with the removal of BinaryProvider from the test dependencies. I'm not sure, though.
Anyway I'd prefer using FFMPEG.jl instead. Then Plots does not have to deal with keeping the links up-to-date as you mentioned in your original post.